lib/crunch-cmd.ML

```diff
@@ -882,9 +892,12 @@ fun crunch_x atts extra prp_name wpigs consts ctxt =
 
         val ctxt'' = ctxt' delsimps simp_dels;
 
-        val (_, thms) = funkyfold (crunch {ctxt = ctxt'', prp_name = prp_name, nmspce = nmspce,
-              wp_rules = wp_rules, wps_rules = wps_rules, igs = igs, simps = simps,
-              ig_dels = ig_dels, rules = rules} pre' extra' [])
+        val crunch_cfg =
+          {ctxt = ctxt'', prp_name = prp_name, nmspce = nmspce, wp_rules = wp_rules,
+           wps_rules = wps_rules, igs = igs, simps = simps, ig_dels = ig_dels,
+           rules = rules}
+        val (_, thms) =
```

Comment:
doesn't the following fit on one line? strange to stair-step it like this

Reply:
True, it does now that the cfg is a separate line.